### ISSUE TITLE
Warning - Template is deprecated

### ISSUE THREAD

**Issue by arsyed04** [valor-software/ngx-bootstrap#1949]
```
Template parse warnings:
The <template> element is deprecated. Use <ng-template> instead ("
  </thead>
  <tbody>
    [WARNING ->]<template ngFor [ngForOf]="rows" let-rowz="$implicit" let-index="index">
      <tr *ngIf="!(datePicke"): ng:///DatepickerModule/DayPickerComponent.html@38:4
```

**Comment by Martin-Luft** [comment]
Please search for same issues before opening a new one...

For Angular 4 just use the @next version which fixes this warning. This cannot be fixed in 1.6.X because it would break Angular 2 support. You can use "ngx-bootstrap": "2.0.0-beta.2" or npm install ngx-bootstrap@next

**Comment by arsyed04** [comment]
Thanks, had searched for open issues instead of closed ones.
Just curious why is this issue only in day picker?
